src/SprykerEco/Yves/Computop/Mapper/Init/AbstractMapper.php

```diff
@@ -78,6 +78,20 @@ protected function generateTransId(TransferInterface $quoteTransfer)
         return md5(implode(self::TRANS_ID_SEPARATOR, $parameters));
     }
 
+    /**
+     * @param \Spryker\Shared\Kernel\Transfer\TransferInterface|\Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return string
+     */
+    protected function generateReqId(TransferInterface $quoteTransfer)
+    {
+        $parameters = [
+            $quoteTransfer->getTotals()->getHash(),
+            $quoteTransfer->getCustomer()->getCustomerReference(),
+        ];
+        return md5(implode(self::TRANS_ID_SEPARATOR, $parameters));
```

Comment:
it's better to use Sha-256 .
Please use https://github.com/spryker/util-text/blob/master/src/Spryker/Service/UtilText/UtilTextService.php to hash things in order to be more abstract.
Maybe even move hashing method to config.
@alex-galych what do you think ?